3 - FightClub/Code/Objects/Fighter.cs

```diff
@@ -0,0 +1,73 @@
+﻿using ObjectOrientedProblems.Code.Enums;
+using ObjectOrientedProblems.Code.Interfaces;
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace FightClub.Code.Objects
+{
+    class Fighter : IFighter
+    {
+        public int Health => _health;
```

Comment:
Though there is nothing wrong with what you did, you can compress a lot of this code by doing:

```
   public int Health { get; private set; } = 10;

		public int Damage { get; private set; } = 1;

		public FighterState State { get; private set; } = FighterState.Healthy;
```